bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -229,7 +235,12 @@ public BatchExecutor(
     LOG.trace("issueRowRequest(Row, Batch.Callback, Object[], index");
     SettableFuture<Object> resultFuture = SettableFuture.create();
     results[index] = null;
-    ListenableFuture<? extends GeneratedMessage> future = issueRequest(row);
+    ListenableFuture<? extends GeneratedMessage> future = null;
+    try {
+      future = issueRequest(row);
+    } catch (InterruptedException e) {
+      future = Futures.immediateFailedFuture(e);
```

Comment:
The Table interface throws InterruptedException.  Propagating the InterruptedException to the user.